django/contrib/admin/templates/admin/delete_confirmation.html

```diff
@@ -20,21 +20,21 @@
 
 {% block content %}
 {% if perms_lacking %}
-    <p>{% blocktrans with escaped_object=object %}Deleting the {{ object_name }} '{{ escaped_object }}' would result in deleting related objects, but your account doesn't have permission to delete the following types of objects:{% endblocktrans %}</p>
+    <p>{% blocktrans with escaped_object=object force_escape %}Deleting the {{ object_name }} '{{ escaped_object }}' would result in deleting related objects, but your account doesn't have permission to delete the following types of objects:{% endblocktrans %}</p>
```

Comment:
Hm, that's obviously an issue. On the other hand, it seems like there's way to use "force_escape" in that fragment to avoid the problem this ticket aims to solve.